syntaxes/omt.tmLanguage.json

```diff
@@ -386,12 +386,29 @@
                     "patterns": [{ "include": "source.odt" }]
                 },
                 {
-                    "match": "(?:\\G|^) +(busyDisabled): *(\\w*)(.*)$",
+                    "match": "(?:\\G|^) +(busyDisabled|blockedIndication|pushmessageIndication): *(\\w*)(.*)$",
                     "captures": {
                         "1": { "name": "keyword.control.omt" },
                         "2": { "patterns": [{ "include": "#_boolean-only" }] },
                         "3": { "patterns": [{ "include": "#_comments-only" }] }
                     }
+                },
+                {
+                    "begin": "^ +(lastEditedBy): +",
+                    "beginCaptures": {
+                        "1": { "name": "keyword.control.omt" }
+                    },
+                    "end": "$",
+                    "contentName": "meta.embedded.iri-literal.odt",
+                    "patterns": [{ "include": "source.odt#_iri-literal" }]
+                },
+                {
+                    "match": "(?:\\G|^) +(startDate|endDate): *(\\d{4}-\\d{2}-\\d{2})(.*)$",
```

Comment:
`startDate` and `endDate` are query type properties, they should be added on line 370.